src/dbnode/runtime/runtime_options.go

```diff
@@ -38,6 +38,11 @@ const (
 	// DefaultBootstrapConsistencyLevel is the default bootstrap consistency level
 	DefaultBootstrapConsistencyLevel = topology.ReadConsistencyLevelMajority
 
+	// DefaultFlushIndexBlockNumSegments is the default number of segments to
+	// attempt to divide a mutable index block into when flushing the index
+	// block.
+	DefaultFlushIndexBlockNumSegments = 8
```

Comment:
Do we want to reduce this back down to 4?

Reply:
sure thing